README.md

```diff
@@ -48,69 +98,375 @@ These bindings allow Azure Functions to integrate with [Azure SignalR Service](h
 1. In a new folder, create a new Azure Functions app.
     - `func init`
 1. Install this Functions extension.
-    - `func extensions install -p Microsoft.Azure.WebJobs.Extensions.SignalRService -v 1.0.0`
+    - `func extensions install -p Microsoft.Azure.WebJobs.Extensions.SignalRService -v 1.0.2`
+
+### Add application settings
+
+App settings in a function app contain global configuration options that affect all functions for that function app. For more details on how to define the app settings, please refer to [App settings reference for Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings)
 
-### Add application setting for SignalR connection string
+An app settings sample for Azure SignalR Service Function binding could be found [here](./samples/simple-chat/csharp/FunctionApp/local.settings.sample.json).
 
-1. Create an app setting called `AzureSignalRConnectionString` with the SignalR connection string.
-    - On localhost, use `local.settings.json`
-    - In Azure, use App Settings
+``` json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "",
+    "AzureWebJobsDashboard": "",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "AzureSignalRConnectionString": "<signalr-connection-string>",
+    "AzureSignalRServiceTransportType": "Transient"
+  },
+  "Host": {
+    "LocalHttpPort": 7071,
+    "CORS": "*"
+  }
+}
+```
+Note that `AzureSignalRConnectionString` and `AzureSignalRServiceTransportType` are settings for Azure SignalR Service Function binding.
+
+* `AzureSignalRConnectionString` is the SignalR Service's connection string, it will be used as default connection string when Function doesn't define it.
+* `AzureSignalRServiceTransportType` is used for determining whether the Function establish WebSockets connections to SignalR Service to send messages.
+  * `Transient`: Send messsages by calling SignalR Service's [REST API](https://github.com/Azure/azure-signalr/blob/dev/docs/rest-api.md). This mode is appropriate when you want to send messages infrequently.
+  * `Persisent`: Establish WebSockets connections for each hub, send all messages via these connections. This mode is more effecient than `Transient` mode when you want to send messages frequently or regularly.
 
 ### Using the SignalRConnectionInfo input binding
```

Comment:
>Using the SignalRConnectionInfo input binding [](start = 4, length = 45)

How about add the [chart](https://github.com/Azure/azure-signalr/blob/dev/docs/images/serverless-arch.png) similar to [https://github.com/Azure/azure-signalr/blob/dev/docs/images/serverless-arch.png](https://github.com/Azure/azure-signalr/blob/dev/docs/images/serverless-arch.png)